pyndl/preprocess.py

```diff
@@ -159,8 +159,19 @@ def create_event_file(corpus_file,
         path where the corpus file is
     event_file : str
         path where the output file will be created
-    symbols : str
-        string of all valid symbols
+    symbols : str or function
+        all valid symbols to include in the events, either as a set of characters or as a 
+        filter function.
+        The set of characters might be explicit or containts Regex character sets. 
+        The function should return `True`, if the passed character is a valid symbol.
+
+        '_', '#', and TAB are special symbols in the event file and might never be included here.
+
+        These examples define the same valid symbols::
+
+            'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
+            'a-zA-Z'
+            lambda chr: ('a' <= chr <= 'z') or ('A' <= chr <= 'Z')
```

Comment:
We should give an example of being as inclusive as possible, like ```lambda character: character not in '#_\t'```

replace `chr` with  `char`, `character` or `symbol`.

Reply:
If someone could craft a regex that has the logic of the "as inclusive as possible" filter function this might give a substantial boost in performance, as I expect the filter function to be relatively slow.

Reply:
I am not sure, whether I fully understand what you mean. Do you want a regex that includes everything except of `'#_\t'`? In that case `[^#_\t]` would be what you are looking for, I assume

Reply:
I might missed it in the discussion, but why do we have 2 parameters for including symbols now?